CHANGELOG.md

```diff
@@ -4,7 +4,7 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
```

Comment:
Please keep the `## [Unreleased]` header for future entries

Reply:
Done: e8e8323

Reply:
We should have both headers for release

Reply:
@wojciechczerniak Shouldn't it be changed on the release branch, not this PR?

Reply:
Up to you guys, I've seen release notes updated here. Just keep them in sync

Reply:
I'm going to be doing the release branch right after this PR is merged, co it doesn't really matter this time 👍